packages/component-model-user/entities/user/resolvers.js

```diff
@@ -0,0 +1,12 @@
+const User = require('./index')
+
+const resolvers = {
```

Comment:
I think this should live at the top level of the component along side `/entities`. See https://gitlab.com/hindawi/xpub/xpub-review/tree/develop/packages/component-model-manuscript/src for context we're not using `src` directories.

Reply:
that does make sense

Reply:
moved to top level in 366dd09